### PR TITLE
Improve example description of "Id attribute value is unique" (3ea0c8)

### PR DESCRIPTION
**Pull request by Jym77**
Improve example descriptions as per #834.

Need for Final Call:
This will not require a Final Call (change to non-normative text)

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/id-value-unique-3ea0c8.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ All `id`s are unique within the document context.
 
 #### Passed Example 3
 
-Two of the `id` are the same (`my-elt`), but they are in different trees (the first one in the [document tree](https://dom.spec.whatwg.org/#document-trees) and the second in the [shadow tree](https://dom.spec.whatwg.org/#shadow-trees)).
+Two of the `id`s are the same (`my-elt`), but they are in different trees (the first one in the [document tree](https://dom.spec.whatwg.org/#document-trees) and the second in the [shadow tree](https://dom.spec.whatwg.org/#shadow-trees)).
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]: I don't think the plural is necessary, as 'id' can be considered implicitly plural.

File: _rules/id-value-unique-3ea0c8.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Two of the `id` are the same (`my-elt`), but they are in different trees (the fi
 
 #### Passed Example 4
 
-Both `id` are the same (`my-elt`), but they are in different [document trees](https://dom.spec.whatwg.org/#document-trees) because the `iframe` is creating a new one.
+Both `id`s are the same (`my-elt`), but they are in different [document trees](https://dom.spec.whatwg.org/#document-trees) because the `iframe` is creating a new one.
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]: As previous comment

File: _rules/id-value-unique-3ea0c8.md (hunk 1 above)
Comment (Jym77) [reply _rules/id-value-unique-3ea0c8.md]: OK. Between @daniel-montalvo (https://github.com/act-rules/act-rules.github.io/pull/1146#pullrequestreview-352979261) and you, I'll trust the native speaker…

File: _rules/id-value-unique-3ea0c8.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ There is only one `id` within the document context.
 
 #### Passed Example 2
 
-All `id`s are unique within the document context.
+All `id` are unique within the document context.
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]:
```suggestion
All `id` attributes are unique within the document context.
```

This is in keeping with #834 (preferred format examples) and #946 (test case description example) to identify code as an element or attribute.

File: _rules/id-value-unique-3ea0c8.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ Both `id` are the same (`my-elt`), but they are in different [document trees](ht
 
 #### Failed Example 1
 
-Several elements have identical `id`.
+The `id` `my-div` is not unique among all `id` in the document.
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]:
```suggestion
The `id` `my-div` is not unique among all `id` attributes in the document.
```

File: _rules/id-value-unique-3ea0c8.md [hunk 5]
```diff
@@ -104,7 +104,7 @@ Several elements have identical `id`.
 
 #### Failed Example 2
 
-Elements of different types have identical `id`.
+The `id` `my-div` is not unique among all `id` in the document.
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]:
```suggestion
The `id` `my-div` is not unique among all `id` attributes in the document.
```

File: _rules/id-value-unique-3ea0c8.md [hunk 6]
```diff
@@ -113,7 +113,7 @@ Elements of different types have identical `id`.
 
 #### Failed Example 3
 
-Having `display: none` on an element still makes it applicable to this rule.
+The `id` `my-div` is not unique among all `id` in the document. This rule still considers element that are neither [included in the accessibility tree][] nor [visible][].
```
Comment (EmmaJP) [inline _rules/id-value-unique-3ea0c8.md]:
```suggestion
The `id` `my-div` is not unique among all `id` attributes in the document. This rule still considers element that are neither [included in the accessibility tree][] nor [visible][].
```

File: _rules/id-value-unique-3ea0c8.md (hunk 3 above)
Comment (Jym77) [reply _rules/id-value-unique-3ea0c8.md]: Good point.

**Pull request merged**